src/validops.cu

```diff
@@ -61,6 +62,29 @@ size_t count_valid_bits_host(std::vector<gdf_valid_type> const & masks, const in
   return count;
 }
 
+
+__global__ 
+void count_valid_bits(valid32_t const * const __restrict__ masks32, const int num_masks32, int * const __restrict__ global_count)
+{
+  typedef cub::BlockReduce<int, block_size> BlockReduce;
+  __shared__ typename BlockReduce::TempStorage temp_storage;
+
+  int cur_mask = threadIdx.x + blockIdx.x * gridDim.x;
+
+  int my_count = 0;
+  while(cur_mask < num_masks32)
```

Comment:
Is `num_masks32` guaranteed to be a multiple of `block_size * gridDim.x`? If not, you could overrun?

Reply:
And if that is expected in the kernel, document the expectation.